configs/fcos/fcos_r50_caffe_fpn_gn-head_1x_bs8x8_coco.py

```diff
@@ -0,0 +1,12 @@
+_base_ = ['./fcos_r50_caffe_fpn_gn-head_1x_coco.py']
+
+data = dict(
+    samples_per_gpu=8,
+    workers_per_gpu=8,
+)
+
+# optimizer
+optimizer = dict(lr=0.04)
+fp16 = dict(loss_scale='dynamic')
+# learning policy
+lr_config = dict(warmup='linear', warmup_iters=1000, warmup_ratio=1.0 / 10)
```

Comment:
Is it necessary to modify warmup_ratio？

Reply:
yes, this modification can enhance the stability of the model training on NPU and avoid the non-convergence in the early stage of training

Reply:
comment added.